src/eduid_common/api/utils.py

```diff
@@ -80,7 +101,7 @@ def save_dashboard_user(user):
     if isinstance(user, User) and not isinstance(user, DashboardUser):
         # turn it into a DashboardUser before saving it in the dashboard private db
         user = DashboardUser(data = user.to_dict())
-    current_app.dashboard_userdb.save(user)
+    getattr(current_app, dbattr_name).save(user)
```

Comment:
Ok you are right. I've removed the argument, and thus I have also removed it from some calls to this function in webapp. You'll shortly receive a small PR for eduid-webapp :)